frontend/Dockerfile.e2e

```diff
@@ -0,0 +1,20 @@
+FROM node:12
+
+RUN mkdir /srv/flagsmith && chown node:node /srv/flagsmith
+
+RUN apt-get update
+RUN wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_91.0.4472.114-1_amd64.deb
```

Comment:
https://github.com/Flagsmith/flagsmith/pull/548/files#diff-88b40629df399d3611837cb436b6d43f743791f1f5944ea002bb76bf3713046aR61

Reply:
After linked PR is merged I think we'd want to update this to match.